Q3.1.py

```diff
@@ -0,0 +1,36 @@
+class Ogrenci:
+    def __init__(self, ogrenciAdi, ogrenciSoyadi, ogrenciSinifi):
+        self.ogrenciAdi = ogrenciAdi
+        self.ogrenciSoyadi = ogrenciSoyadi
+        self.ogrenciSinifi = ogrenciSinifi
+
+    def ogrenciBilgiGoster(self):
+        print("Adı : {} \n Soyadı : {} \n Sınıfı : {}".format(self.ogrenciAdi, self.ogrenciSoyadi, self.ogrenciSinifi))
+
+
+class Soru:
+    def __init__(self):
+        self.dogruSayisi = int(input("Öğrenci doğru sayısı :"))

```

Comment:
değişken isimlerini dogru_sayisi şeklinde değiştirebilir misin? fonksiyon isimleri de camel case olduğu için karışıklık çıkabilir